cves/kernel/CVE-2021-27363.yml

```diff
@@ -177,8 +178,8 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: "nothing was said"
```

Comment:
Nothing was said by the developers, but would it be feasible? Why or why not? What sources did you check? Any additional information on this aspect would be nice.

Reply:
I must have interpreted this question wrong as well. I doubt an automated tool would have discovered this but I will look into that once again just to back up my conclusion.